mevolib/fetch/BioSeqs.py

```diff
@@ -367,174 +304,151 @@ def from_entrez (cls, email, entrez_db, query, max_fetch = sys.maxsize):
         report = [(date_time, 'entrez', entrez_db, query)]
         return (cls(seq_dict, report))
 
-    def __len__ (self):
-        """
-        Returns:
-            int
-                Total number of sequences.
+
+    def __len__ (self) -> int:
+        """Total number of sequences.
         """
         return (len(self.data))
 
-    def __str__ (self):
-        """
-        Returns:
-            string
-                Fancy print of the data stored in the BioSeqs object.
+
+    def __str__ (self) -> str:
+        """Fancy print of the data stored in the BioSeqs object.
```

Comment:
```suggestion
        """Fancy print of the data stored in the BioSeqs object."""
```
And remove next `"""`.